lib/utils/ttyname.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I'd move this to a dedicated package. `utils` is usually the last place to implement anything - unless you cannot put in into any other domain.